app/views/incident_reports/_form.html.haml

```diff
@@ -244,6 +244,32 @@
         prompt: true
   .passenger-incident-info{style: ('display: none;' unless report.passenger_incident?)}
     %h3 Passenger Incident Information
+    .field
+      = form.label :inj_pax_info
+      = form.check_box :inj_pax_info, checked: report.has_injured_passengers?,
+        id: :supervisor_report_inj_pax_info
+    .inj-pax-info{style: ('display: none;' unless report.has_injured_passengers?)}
+      %h3 Injured Passenger Info
```

Comment:
So, I didn't fill anything in on the Driver Account of Incident. I just submitted it, and as expected, errors were displayed. However, when I checked `Did the incident involve a passenger?`, `Were passengers injured?`, left `Injured Passenger Info` blank, and clicked `Save report and print PDF`, the errors don't appear anymore.

Didn't look too far into it, just wanted to let you know that this doesn't seem like expected behavior to me. Could be wrong.

Reply:
Something strange is going on. In my environment, when I enter any passenger info and submit the form, the submit button is no longer clickable. Pressing Enter in a text field doesn't submit the form. Will investigate more tomorrow.

Reply:
Fixed. https://github.com/umts/incidents/pull/128/commits/04dc98fab1b12cacbac39379aa6b5b9d39ddcce4

Reply:
Yup, looks good.